### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by lucab85**
Fix typos

### CODE REVIEW

**Comment by lucab85** [comment]
Thank you @jpmedley. Please help me with the failure of Travis-CI.

**Comment by PaulKinlan** [comment]
I believe the travis failure is because your branch is not on the main repo and we have issues testing them here.  This PR all looks entirely reasonable.

**Comment by PaulKinlan** [comment]
(thank you for the PR.)

File: src/content/en/ilt/pwa/e-commerce-lab-3-payment-request-api.md [hunk 1]
```diff
@@ -442,7 +442,7 @@ request.addEventListener('shippingoptionchange', e => {
 
 #### Explanation
 
-Upon receiving the `shippingoptionchange` event, the `request` object's `shippingOption` is updated. The `shippingOption`It indicates the `id` of the selected shipping options. The `id` is passed to `buildPaymentsDetails`, which looksLook for the price of the shipping option and updates the display items so that the user knows the total cost is changed. `buildPaymentsDetails alsoAlso` changes the shipping option's `selected` property to `true` to indicate that the user has chosen the item. Checkout `buildPaymentDetails()` function in __app/scripts/modules/payment-api.js__ to see how it works.
```
Comment (petele) [inline src/content/en/ilt/pwa/e-commerce-lab-3-payment-request-api.md]: This file is ingested from a GoogleDoc source, so changes will be lost on our next update.

@DavidScales, @nasearle, @agektmr: fyi to update the original doc

File: src/content/en/ilt/pwa/live-data-in-the-service-worker.md [hunk 2]
```diff
@@ -110,7 +110,7 @@ Different browsers allow different amounts of offline storage. This table summar
 
 ### Storing data with IndexedDB
 
-IndexedDB is a noSQL database. IndexedDB data is stored as key-value pairs in __object stores__. The table below shows an example of an object store, in this case containing beverage items:
```
Comment (petele) [inline src/content/en/ilt/pwa/live-data-in-the-service-worker.md]: This file is ingested from a GoogleDoc source, so changes will be lost on our next update.

@DavidScales, @nasearle, @agektmr: fyi to update the original doc

File: src/content/en/ilt/pwa/working-with-indexeddb.md [hunk 3]
```diff
@@ -31,7 +31,7 @@ This text guides you through the basics of the  [IndexedDB API](https://develope
 
 
 
-IndexedDB is a large-scale, noSQL storage system. It lets you store just about anything in the user's browser. In addition to the usual search, get, and put actions, IndexedDB also supports transactions. Here is the definition of IndexedDB on MDN:
+IndexedDB is a large-scale, NoSQL storage system. It lets you store just about anything in the user's browser. In addition to the usual search, get, and put actions, IndexedDB also supports transactions. Here is the definition of IndexedDB on MDN:
```
Comment (petele) [inline src/content/en/ilt/pwa/working-with-indexeddb.md]: This file is ingested from a GoogleDoc source, so changes will be lost on our next update.

@DavidScales, @nasearle, @agektmr: fyi to update the original doc

**Pull request merged**